src/OVAL/probes/probe/icache.c

```diff
@@ -39,11 +39,14 @@
 #include "probe-api.h"
 #include "common/debug_priv.h"
 #include "common/memusage.h"
+#include "oscap_helpers.h"
 
 #include "probe.h"
 #include "icache.h"
 #include "_sexp-ID.h"
 
+#define PROBE_ITEM_COLLECT_MAX 1000
```

Comment:
Here was used `1.000` but in the commit description is mentioned `10.000`. Also, do you have information about how much memory one item consumes? I would like to understand how this proposed limit was calculated.